### DISCUSSION TITLE
Display component as table first row

### DISCUSSION THREAD

**Discussion by FRicardi** [UI5/webcomponents-react#4508] [Q&A] [answered]
Hello!

My use case is that I want to show inside a table, before the first row, a message with selected filters. As per this [code sandbox](https://codesandbox.io/s/table-render-a-component-below-header-fl0sxc?file=/src/App.js), what I've found is that is possible to render something else than a `TableCell`, thought it will only be displayed on the first column. 

Is there a way to render an entire component as the first row? It can be on a regular table or `AnalyticalTable`, any of those would solve my problem.

**Comment by FRicardi** [comment]
This behavior is not relevant for my use case, closing the discussion.

**Comment by MarcusNotheis** [comment, marked as answer]
Hi @FRicardi,

unfortunately this is not possible as both the `Table` and the `AnalyticalTable` expect all rows to contain as many cells as your table has columns.
For your `Filtered by...` example in the codesandbox, I would recommend to render a `ToolBar` or a `FilterBar` above the respective table where you can display the required information.
